jekyll/_layouts/classic-docs.html

```diff
@@ -41,13 +41,17 @@
 
             <div class="quick-info-list">
               <time aria-describedby="tooltip-time" id="time-posted-on" datetime="{{  page.last_modified_at | date_to_xmlschema }}">
-                <img src="{{ site.baseurl }}/assets/img/docs/duration.svg">
-                {% if page.last_modified_at %}<span id="time-posted-ago"></span>{% else %} {{ page.date | date: '%d-%b-%y' }} {% endif %}
+                <img alt="Clock" src="{{ site.baseurl }}/assets/img/docs/duration.svg">
+                  {% assign dateStart = page.last_modified_at | date: '%s' %}
+                  {% assign nowTimestamp = 'now' | date: '%s' %}  
+                  {% assign timestampDiff = nowTimestamp | minus: dateStart %}
+                  {% assign yearInSeconds = 60 | times: 60 | times: 24 | times: 365 %}
+                  {% if timestampDiff > yearInSeconds %} 1+ year ago {% else %} {{ page.last_modified_at | timeago }} {% endif %}
```

Comment:
There are extra spaces on both sides of the string. Can we please fix that?
![Screen Shot 2022-01-07 at 12 53 36 PM](https://user-images.githubusercontent.com/1449325/148606153-e35a7f9f-bdd0-4073-ae97-4b2d4242aadc.png)